CONTRIBUTING.md

```diff
@@ -2,59 +2,64 @@
-
-## Submitting a Pull Request
-#### Proposing a Change
-1. **Before writing any code**, please post a proposal of the solution you plan to implement as a comment in the GH issue. This should include a brief technical explanation of the changes you will make.
-1. Wait for Expensify to review and provide feedback on the proposal within the GH issue, and assign the GH issue to you. Please do not move forward with creating a Pull Request until your proposal is approved
-1. Fork this repository and create a new branch
-1. ‼️ **Before you start writing any code** ️‼️, please be aware that we require all commits to be [signed](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/signing-commits). The easiest way to do that is to [generate a new GPG key](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-gpg-key) and [add it to your Github account](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/adding-a-new-gpg-key-to-your-github-account). Once you've done that, you can automatically sign all your commits by adding the following to your `.gitconfig`:
-    ```
-    [commit]
-        gpgsign = true
-    [user]
-        email = <Your GH account email>
-        name = <Your Name>
-        signingkey = <your_signing_key>
-    [gpg]
-        program = gpg
-    ```
-1. [Open a PR](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork). Be sure to fill in all the required information on the PR template.
-1. An Expensify engineer will be automatically assigned to review your PR
-1. You will need all checks to pass:
-	1. CLA - You must sign our [Contributor License Agreement](https://github.com/Expensify/Expensify.cash/blob/master/CLA.md) by following the CLA bot instructions that will be posted on your PR
-	1. Tests - All tests must pass before a merge of a pull request
-	1. Lint - All code must pass lint checks before a merge of a pull request
-
-#### Testing
-Upon submission of a PR, please include a numbered list of explicit testing steps for each platform (Web, Desktop, iOS, and Android) to confirm the fix works as expected and there are no regressions.
-
-#### Review Tips
-
-### Getting Attention
-- If you have made a change and are ready for another review the best way to get your reviewer's attention is to leave a comment that says "Updated" on the PR itself.
-- Please do NOT ping individual reviewers in the Slack channel to get their attention and keep the conversation in GitHub whenever possible.
-- Patience is a virtue. Reviews can sometimes take place over the course of several days. If your PR has not been addressed after 3 days please leave a comment on the PR. If the PR goes without a response for another day please let us know via Slack.
-
-### JavaScript Style
```

Comment:
Coming from https://expensify.slack.com/archives/C011W8BJ9L6/p1615481353152800?thread_ts=1615345614.094600&cid=C011W8BJ9L6, let's keep this section here